packages/nimble-components/src/table/specs/table-column-specs/table-column-elapsed-time-hld.md

```diff
@@ -0,0 +1,177 @@
+# Elapsed Time Table Column
+
+## Overview
+
+The elapsed time table column will provide a way to visualize numeric data that represents a timespan. A 'timespan' is simply a numeric value representing a total number of seconds (i.e. a value of 5 means '5 seconds').
+
+### Background
+
+[Work Item](https://ni.visualstudio.com/DevCentral/_workitems/edit/2526116)
+
+### Non-goals
+
+-   APIs to configure individual aspects of the display (i.e. fractional digits for seconds display)
+-   APIs to format the elapsed time in a variety of ways
+    -   Show elapsed time as digital display (00:01:20.323)
+    -   Show elapsed time with symbols indicating a quantitative value (i.e. "< 1 sec")
+
+### Risks and Challenges
+
+Formatting the elapsed time units in the exact way that SLE presents them would require custom logic that differs from what the browser provides natively using APIs like `Intl.Numberformat.format()`. The preference here is to align with current browser behaviors for unit formatting, which includes localization.
+
+### Prior Art/Examples
+
+![ElapsedTime SLE](../spec-images/ElapsedTimeSLE.PNG)
+
+## Design
+
+Below is an example of how the `nimble-table-column-elapsed-time` would be used within a `nimble-table`:
+
+```HTML
+<nimble-table>
+    <nimble-table-column-elapsed-time field-name="elapsedTime">Elapsed Time</nimble-table-column-elapsed-time>
+</nimble-table>
+```
+
+The displayed value for elapsed time will always take the form of "\<hour display>, \<minute display>, \<second display>.\<fractional seconds>", where the commas are non-localized delimiters <sup>1</sup>.
```

Comment:
Unless I missed it, I didn't see anything that stated how many `fractional seconds` would be shown. I assume it isn't configurable since I didn't see an API to control that. Are you thinking that it'll be like currently implemented in SLE, which is a max of 2 digits but could be less if not necessary for a given value (i.e. `maximumFractionDigits: 2` on the [NumberFormatter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat))?

Reply:
I made mention of it in the paragraph in the 'API' section. To summarize I was thinking we would just default to showing at most 2 fractional digits, just like in SLE today, and forgo the configurable API for now.